dists/scummvm.appdata.xml

```diff
@@ -6,29 +6,258 @@
   <project_license>GPL-3.0-or-later</project_license>
   <name>ScummVM</name>
   <summary>Interpreter for numerous adventure games and role-playing games</summary>
+  <summary xml:lang="ru">Интерпретатор множества приключенческих и ролевых игр</summary>
```

Comment:
These, and other language tags should be sorted alphabetically, so their sequence stays consistent

Reply:
Fixed